frontend/public/components/notification-drawer.tsx

```diff
@@ -169,6 +156,41 @@ export const ConnectedNotificationDrawer_: React.FC<ConnectedNotificationDrawerP
     }
   }, [criticalAlertList, isAlertExpanded, isDrawerExpanded, prevDrawerToggleState]);
 
+  const emptyState = !_.isEmpty(loadError) ? (
```

Comment:
Though I'm not a fan of inlining the `EmptyState` right inside the `ConnectedNotificationDrawer_` component, I'd rather pluck it out and call it.  Tends to get messy as things grow.

Reply:
@spadgett updated the design as noted above. thanks

![Screen Shot 2020-02-03 at 4 56 45 PM](https://user-images.githubusercontent.com/35978579/73694675-8c43f200-46a6-11ea-8b9b-6e0d55aba433.png)

Reply:
> Though I'm not a fan of inlining the `EmptyState` right inside the `ConnectedNotificationDrawer_` component, I'd rather pluck it out and call it. Tends to get messy as things grow.

It depends on `loadError` and `toggleNotificationDrawer`, which are local to the function, and is only used in this component.

Reply:
If we pull it out, we should make these both real components that take props instead of functions. But it's not too large, so I'm OK leaving it inline.